pkg/node/manager.go

```diff
@@ -72,7 +72,8 @@ func UpdateNode(ni Identity, n *Node) {
 
 	nodeIP := n.GetNodeIP(false)
 	updateNodeCIDR(n.IPv4AllocCIDR, nodeIP)
-	updateNodeCIDR(n.IPv6AllocCIDR, nodeIP)
+	nodeIP6 := n.GetNodeIP(true)
+	updateNodeCIDR(n.IPv6AllocCIDR, nodeIP6)
```

Comment:
I think we need to drop this altogether for now until the data path supports encapsulation over IPV6. It's questionable to me whether that is even needed because if someone runs  IPv6 on the underlay network, they would definitely want to run natively without encap.